### ISSUE TITLE
Can't change text colors

### ISSUE THREAD

**Issue by artonragsdale** [joachimesque/Bulma.sketch#1]
I'm using Sketch version 49 and macOS 10.13.3.

All text is shows up as black, regardless of type setting. When I try to change to a text setting with a different color, the text value changes to the modifier (e.g. is-size-4) but the color stays black.

**Comment by joachimesque** [comment]
You're right, that was a bug! I don't know how it came to be, but the new version is all right.
